packages/fxa-auth-server/lib/routes/recovery-phone.ts

```diff
@@ -129,6 +160,18 @@ export const recoveryPhoneRoutes = (
         return recoveryPhoneHandler.confirm(request);
       },
     },
+    {
+      method: 'POST',
+      path: '/recovery-phone/send_code',
+      options: {
```

Comment:
Do we need a validate check on the phoneNumber being sent in the payload?

Reply:
@dschom Don't believe so, this api should send a code to the phone number created by the create api.

Reply:
Sorry, that was a misread on my part.